main.go

```diff
@@ -14,11 +14,12 @@ import (
 )
 
 var Options struct {
-	AssistedServiceURL string `envconfig:"ASSISTED_SERVICE_URL" default:"http://assisted-service:8080"`
-	Port               string `envconfig:"PORT" default:"8080"`
-	HTTPSKeyFile       string `envconfig:"HTTPS_KEY_FILE"`
-	HTTPSCertFile      string `envconfig:"HTTPS_CERT_FILE"`
-	DataDir            string `envconfig:"DATA_DIR"`
+	AssistedServiceScheme string `envconfig:"ASSISTED_SERVICE_SCHEME"`
```

Comment:
I guess it's easier to split it here then via regex later?

Reply:
or via the urllib tools.

Reply:
Yeah, I'm actually split on this. Theoretically I could make this entire thing some kind of url template.

Then it would go something like:
1. Replace `{image_id}` in the template with our request it
2. Parse the url out
3. Add other info to url (auth, etc.)
4. Use that for request

The more I think about it the more I want to go this direction. Do you think it's worth doing that in this patch now?

Reply:
That all said I think I misunderstood your initial question.
I split out the host and scheme so that I didn't have to parse them out and then recombine them later.

I could do it that way though if you would rather prioritize reducing the number of env vars. I don't feel super strongly about it either way.